### DISCUSSION TITLE
How to use pandoc-server on MacOS

### DISCUSSION THREAD

**Discussion by guanpengchn** [jgm/pandoc#9843] [Q&A] [answered]
I have installed pandoc. And run pandoc-server like this:

<img width="576" alt="image" src="https://github.com/jgm/pandoc/assets/13995641/03f01f8a-0036-4449-835f-71c7683fa7b6">

When I curl this port, there is nothing happened. Why?

<img width="566" alt="image" src="https://github.com/jgm/pandoc/assets/13995641/0db96e1e-9c76-42be-b55c-6b98b229e854">

I also try it on Linux server. But the same answer.

**Comment by tarleb** [comment, marked as answer]
The correct invocation would be

``` sh
curl -H 'Content-Type: application/json' \
    --data-raw '{"from":"html", "text":"<h1>My headline</h1>","to":"markdown"}' \
    --X POST \
    http://localhost:3030/
```

See the [pandoc-server manpage](https://pandoc.org/pandoc-server.html).

**Comment by guanpengchn** [comment]
I know. Thank you. 

Server has no log.

I try this code has result.

```
curl -H 'Content-Type: application/json' \
    --data-raw '{"from":"html", "text":"<h1>My headline</h1>","to":"markdown"}' \
    -X POST \
    http://localhost:3030/
```

<img width="1157" alt="image" src="https://github.com/jgm/pandoc/assets/13995641/1a91b1b6-da93-49b5-9c0f-ec007ba0564f">
